.travis.yml

```diff
@@ -212,7 +212,7 @@ matrix:
     compiler: gcc
     env:
       - COMPILER=g++-8
-      - CXXFLAGS=-std=c++17
+      - CMAKE_OPTIONS=-DJSON_ForceCpp17=ON
```

Comment:
Should this flag also be used in AppVeyor?

Reply:
The AppVeyor cfg currently uses `/std:c++latest`. Imo the new flag is more descriptive, but maybe I'm biased. :-) Shall I change the AppVeyor cfg as well? I think there's nothing to be fixed, but it would be more consistent, I guess.

Reply:
Right, then I think we should leave it as is.